src/components/Molecules/LanguageMenu/LanguageMenu.tsx

```diff
@@ -58,7 +60,9 @@ export const LanguageMenu: FC<LanguageMenuProps> = ({
     }
   })
 
-  const myLanguageContent = <LanguageMenuList languages={languages} onClick={handleLanguageClick} />
+  const myLanguageContent = (
+    <LanguageMenuList languages={languages} onClick={handleLanguageClick} languageId={languageId} />
```

Comment:
it's a index as opposed to an ID 👍 
```suggestion
    <LanguageMenuList languages={languages} onClick={handleLanguageClick} currentIndex={currentIndex} />
```